### PR TITLE
feat: export `metadata` object in local stage 2

### PR DESCRIPTION
**Pull request by eduardoboucas**
Same as #122 but for the stage 2 file used in local development.

### CODE REVIEW

File: test/node/stage_2.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  // not supported. The best we can do is to naively look for some assignments.
+  if (semver.lt(process.version.slice(1), '14.0.0')) {
+    for (const func of functions) {
+      t.true(stage2.includes(`metadata["${func.name}"] = {"url":"${pathToFileURL(func.path).toString()}"}`))
+    }
+
+    return
+  }
+
+  const { stdout, stderr } = await execa('node', [stage2Path])
```
Comment (mraerino) [inline test/node/stage_2.ts:64]: my understanding would be that the code you are generating runs in deno? why are you executing node here?

Comment (eduardoboucas) [reply test/node/stage_2.ts:64]: This code runs in Node as part of local development, called by the CLI.

Comment (eduardoboucas) [reply test/node/stage_2.ts:64]: Called here: https://github.com/netlify/cli/blob/main/src/lib/edge-functions/proxy.js#L166-L180

Comment (eduardoboucas) [reply test/node/stage_2.ts:64]: Hang on, I'm holding this wrong! This is called from Node but then actually runs in Deno. 🤯 

Let me change this.

Comment (eduardoboucas) [reply test/node/stage_2.ts]: Done in https://github.com/netlify/edge-bundler/pull/126/commits/4fcd2c190c0aee5272029128b3b5b7e7363548a2. Thanks!

**Review by mraerino — APPROVED** [review]

**Pull request merged**